README.md

```
@@ -4,6 +4,7 @@
 
 OliveTin gives **safe** and **simple** access to predefined shell commands from a web interface.
 
+[![Static Badge](https://img.shields.io/badge/maturity-stable-brightgreen)](#none)
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)